mindsdb/integrations/handlers/sendinblue_handler/sendinblue_tables.py

```diff
@@ -58,4 +60,50 @@ def get_email_campaigns(self, **kwargs):
         connection = self.handler.connect()
         email_campaigns_api_instance = sib_api_v3_sdk.EmailCampaignsApi(connection)
         email_campaigns = email_campaigns_api_instance.get_email_campaigns(**kwargs)
-        return [email_campaign for  email_campaign in email_campaigns.campaigns]
\ No newline at end of file
+        return [email_campaign for  email_campaign in email_campaigns.campaigns]
+              
+    def delete(self, query: ast.Delete) -> None:
+        """
+        Deletes an email campaign from Sendinblue.
+
+        Parameters
+        ----------
+        query : ast.Delete
+           Given SQL DELETE query
+
+        Returns
+        -------
+        None
+
+        Raises
+        ------
+        ApiException
```

Comment:
Let's correct the type of Exception that is raised here based on your recent change.